src/OpenApiValidation/Helpers/Schema.php

```diff
@@ -27,7 +27,9 @@ public static function openApiToJsonSchema(array $schema) : array
         }
         if ($schema['nullable'] ?? false) {
             unset($schema['nullable']);
-            $schema['type'] = [$schema['type'], 'null'];
+            if (isset($schema['type'])) {
```

Comment:
Without this check, the `anyOf`, `oneOf`, etc schemas will trigger a warning about `$schema['type']` not being defined.